keystorage/azure/src/main/java/tech/pegasys/signers/azure/AzureKeyVault.java

```diff
@@ -83,12 +93,12 @@ public CryptographyClient fetchKey(final String keyName, final String keyVersion
     final String keyId = key.getId();
 
     return new CryptographyClientBuilder()
-        .credential(clientSecretCredential)
+        .credential(tokenCredential)
         .keyIdentifier(keyId)
         .buildClient();
   }
 
-  public static String constructAzureKeyVaultUrl(final String keyVaultName) {
+  private static String constructAzureKeyVaultUrl(final String keyVaultName) {
```

Comment:
leaving this public makes it easier to use from other places - and given it doesn't expose anything "dangerous" internal to the class, better to make it open (IMHO).

Reply:
reverted.